app/DTO/Slot.php

```diff
@@ -14,57 +14,46 @@
 
 final class Slot
 {
-    private string $publicKey;
-
-    private int $order;
-
-    private WalletViewModel $wallet;
-
-    private Carbon $forgingAt;
-
-    private array $lastBlock;
-
-    private string $status;
-
-    private int $currentRoundBlocks;
+    private array $collection;
```

Comment:
The changes look good to me I am just not sure about using `collection` as the name, maybe something like state or something different, what do you think 🤔